cg/store/status_customer_filters.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+from typing import List, Optional, Callable
+
+from sqlalchemy.orm import Query
+
+from cg.store.models import Customer
+
+
+def filter_customer_by_customer_id(customers: Query, customer_id: str, **kwargs) -> Query:
```

Comment:
I think this should be customer_internal_id 
customer_id in a different context can refer to the customer index and not internal_id which can be confusing

Reply:
Agreed - Fixed